test/lib/slots.test.ts

```diff
@@ -0,0 +1,56 @@
+import getSlots from '@lib/slots';
+import {it, expect} from '@jest/globals';
+import MockDate from 'mockdate';
+import dayjs, {Dayjs} from 'dayjs';
+import utc from 'dayjs/plugin/utc';
+import timezone from 'dayjs/plugin/timezone';
```

Comment:
Codacy has a fix for the issue: [Replace `'dayjs/plugin/timezone'` with `"dayjs/plugin/timezone"`](https://app.codacy.com/gh/calendso/calendso/pullRequest?prid=7689306)
```suggestion
import timezone from "dayjs/plugin/timezone";
```